packages/eas-cli/package.json

```diff
@@ -58,6 +58,7 @@
     "indent-string": "4.0.0",
     "jks-js": "1.1.0",
     "joi": "17.11.0",
+    "js-yaml": "^4.1.0",
```

Comment:
FWIW in other projects we use [`yaml`](https://www.npmjs.com/package/yaml), not sure why. I guess it has the benefit of being more actively maintained? Would you suggest we move to `js-yaml`, should this move to `yaml` or do you expect it's ok to keep two separate YAML parsers?